Circle.cpp

```diff
@@ -1,14 +1,28 @@
 #ifndef Circle_cpp
 #define Circle_cpp
+
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
+
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape
 {
 public:
     void drawShape() override
     {
-        cout << "circle" << endl;
+        cout << "circle" << endl; 
+    }
+    int inputAndCalculateArea() override
+    {
+        int Radius, area;
+        int PI = 3.14;
```

Comment:
PI is a global constant, either use `M_PI` from `math.h`. Or define it as a Macro yourself.